site/content/installation/installing-ngf/manifests.md

```diff
@@ -238,7 +233,7 @@ Follow these steps to uninstall NGINX Gateway Fabric and Gateway API from your K
      ```
 
      ```shell
-     kubectl delete -f https://github.com/nginxinc/nginx-gateway-fabric/releases/download/v1.2.0/crds.yaml
+     kubectl delete -f https://raw.githubusercontent.com/nginxinc/nginx-gateway-fabric/v1.2.0/deploy/crds.yaml
```

Comment:
Can you also update the uninstall instructions in the helm documentation:

-  https://github.com/nginxinc/nginx-gateway-fabric/tree/main/charts/nginx-gateway-fabric#uninstalling-the-chart
- https://github.com/nginxinc/nginx-gateway-fabric/blob/main/site/content/installation/installing-ngf/helm.md#uninstall-nginx-gateway-fabric

Reply:
Not sure what version of the CRDs we should use there...I think for the website we can use the tag, but not sure about the chart docs. 
What happens if somebody tries to uninstall a different version of a CRD? Does it throw an error?

Reply:
We can't just use the main branch for the chart docs? 

> What happens if somebody tries to uninstall a different version of a CRD? Does it throw an error?

Just tried it out and didn't get any errors. It just deleted the crd

Reply:
Yeah I'm good with using main

Reply:
Upgrade command in there should also be updated. Issue with pointing to the tag right now is that it doesn't exist for 1.2. Going forward it should be fine.

If they install a different version of the CRDs then there could be errors in the NGF logs, but not sure yet what the behavior will be.

Reply:
Uninstalling shouldn't be an issue, but upgrading could be.

Reply:
Ah, yeah since we require them to pull the chart, then I think that's fine as-is, since it'll be the correct version.